LICENSE

```diff
@@ -1,6 +1,192 @@
-   Salt - Remote execution system
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
 
-   Copyright 2014-2016 SaltStack Team
```

Comment:
If this gets pulled it needs to be re-added, probably down in L189.

Reply:
I have to defer to @thatch45 here. I suspect that moving this to the README is fine but I'm not completely sure.

Reply:
sorry for the delay, lets change it to SaltStack Team, lets not risk making this a legal event :)

Reply:
@eirinikos Did you see the reply above?